backend/config/logback.xml

```diff
@@ -3,14 +3,56 @@
     <include resource="org/springframework/boot/logging/logback/defaults.xml"/>
     <include resource="org/springframework/boot/logging/logback/file-appender.xml"/>
 
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
+            <Pattern>
+                %d %clr(%-5p) %clr(%-35logger{5}){cyan} - %-60msg%n
+            </Pattern>
+        </encoder>
+    </appender>
+
+    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+        <file>log/backend.log</file>
+        <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
+            <Pattern>
+                %d %clr(%-5p) %clr(%-35logger{5}){cyan} - %-60msg%n
+            </Pattern>
+        </encoder>
+    </appender>
+
+    <appender name="SYSLOG" class="ch.qos.logback.classic.net.SyslogAppender">
+        <!--Need to change this-->
+        <syslogHost>127.0.0.1</syslogHost>
```

Comment:
Regarding this, do we need to change this to something else? If so, what should it be? Same, for pss/config/logback.xml

Reply:
In the committed files this should not change. Do add the instructions on how to do localhost testing. For the deployed files we (you :) will need to coordinate with networking team to figure out where to send syslog events, and explain to them what syslog entries our operators / engineers will be seeing.